src/lib.rs

```diff
@@ -665,6 +667,36 @@ impl Build {
         self
     }
 
+    /// Specify the C or C++ language standard version.
+    ///
+    /// Common values accepted by modern versions of GCC, Clang and MSVC:
+    /// - `c11` for ISO/IEC 9899:2011
+    /// - `c17` for ISO/IEC 9899:2018
+    /// - `c++14` for ISO/IEC 14882:2014
+    /// - `c++17` for ISO/IEC 14882:2017
+    /// - `c++20` for ISO/IEC 14882:2020
+    ///
+    /// Other values like `c99`, `gnu89` and `c++11` have less broad support.
```

Comment:
It should be noted that this isn't entirely true: `c++11` has the *most* broad support of any standard listed here across the greatest variety of compilers and platforms.

Reply:
Maybe I should have been more direct here: [`c99`, `gnu89` and `c++11`](https://learn.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version?view=msvc-170) are not supported by the `/std:` flag to MSVC.

Reply:
Even Visual Studio 2015 doesn't support `c++11` as a language standard flag, if the documentation is to be believed. The minimum is `c++14`.

For the same reasons, I mention `c99` as less widely supported, because MSVC has never supported C99, only C11. That's because C11 made some features optional that were mandatory in C99.

Microsoft also don't support the GNU extensions, but that's less surprising.